sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -246,4 +246,19 @@ void flash_ctrl_testutils_increment_counter(dif_flash_ctrl_state_t *flash_state,
                                             uint32_t *strike_counter,
                                             uint32_t index);
 
+/**
+ * Sets a strike counter to an arbitrary value.
+ *
+ * If the arbitrary value is 0 or is impossible (attempts to flip a bit
```

Comment:
I think it should be perfectly fine to clear the counter.

Reply:
yeah you're right, 0 should be allowed, similar to comment above.